projects/common/src/constants/application-constants.ts

```diff
@@ -1,4 +1,5 @@
 export const enum ApplicationFeature {
   PageTimeRange = 'ui.page-time-range',
-  FeatureDefaultTimeRangeMap = 'ui.feature-default-time-range-map'
+  FeatureDefaultTimeRangeMap = 'ui.feature-default-time-range-map',
+  EnableTriggerBasedSearch = 'ui.enable-trigger-based-search'
```

Comment:
only question here is whether to remove `enable` from ff name/key for consistency.

Reply:
updated